### ISSUE TITLE
rancher container not running anymore - log output: no space left on device - but there is more than enough space left

### ISSUE THREAD

**Issue by pogii123** [rancher/rancher#28006]
**What kind of request is this (question/bug/enhancement/feature request):**
question

**Steps to reproduce the issue**
Run a rancher cluster with 1 master and 2 workers
After like 5 Months of running fine the master rancher/rancher container stops working

**Result:**
The rancher/rancher container has stopped working and is restarting.
The container logs are as follow
2020-07-21 20:00:05.870984 I | raft: 8e9e05c52164694d became leader at term 561
2020-07-21 20:00:05.871006 I | raft: raft.node: 8e9e05c52164694d elected leader 8e9e05c52164694d at term 561
2020-07-21 20:00:05.872609 I | etcdserver: published {Name:default ClientURLs:[http://localhost:2379]} to cluster cdf818194e3a8c32
2020-07-21 20:00:05.872966 I | embed: ready to serve client requests
2020-07-21 20:00:05.874672 N | embed: serving insecure client requests on 127.0.0.1:2379, this is strongly discouraged!
I0721 20:00:05.888686       6 http.go:116] HTTP2 has been explicitly disabled
log: exiting because of error: log: cannot create log: open /tmp/rancher.98ec3b8824c3.root.log.INFO.20200721-200005.6: no space left on device

So basically it says that there is no space left on device. But I checked every connected device.
NFS Server has like 90GB free space
/var has 6GB left
docker thin pool has 30 GB left

**Other details that may be helpful:**
/var had only 600MB left and I extended the partition for about 6GB
after that I restarted container and after it was still not working I rebooted the server
I also did a docker image prune -a and cleared 6GB of Images. 

The applications on the Kubernetes/rancher cluster are still running and accessible. Only the fronted and the cluster itself is not accessible

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
- Installation option (single install/HA):
rancher/rancher:latest 
HA installation 

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Self Hosted Cluster on CentOS 7.7 Hosts
- Machine type (cloud/VM/metal) and specifications (CPU/memory): vm each 4CPU/16GB RAM
- Kubernetes version (use `kubectl version`):

```
The connection to the server XXX was refused - did you specify the right host or port?
kubectl is currently not working. Because the cluster is not accessible
```

- Docker version (use `docker version`):

```
~]# docker version
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea
 Built:             Wed Nov 13 07:25:41 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.5
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.12
  Git commit:       633a0ea
  Built:            Wed Nov 13 07:24:18 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

**Comment by vincent99** [comment]
You're probably out of inodes; `df -i`

**Comment by pogii123** [comment]
@vincent99 good idea, I checked it.. But non of my devices are using more than 10% of their inodes.

**Comment by pogii123** [comment]
Found the full volume. It was a Block device which was at 100% usage but when mounted locally it was like nearly empty.

~]# docker exec -ti objective_wilbur df -h
Filesystem                                                                                         Size  Used Avail Use% Mounted on
/dev/mapper/docker-253:9-2349538-43691ae8945c066b08c2a410869f9138dfbdf2a1df867d3c1ffe54aa6d481f6c   10G   10G   24K 100% /
tmpfs                                                                                               64M     0   64M   0% /dev
tmpfs                                                                                              7.8G     0  7.8G   0% /sys/fs/cgroup
shm                                                                                                 64M     0   64M   0% /dev/shm
/dev/mapper/vgsystem-lvvar                                                                         8.0G  2.5G  5.6G  31% /etc/hosts
tmpfs                                                                                              7.8G     0  7.8G   0% /proc/acpi
tmpfs                                                                                              7.8G     0  7.8G   0% /proc/scsi
tmpfs                                                                                              7.8G     0  7.8G   0% /sys/firmware


I've deleted some old log files and now its running again.

Now I will search for something like log rotate inside of rancher..
